test/enum_test.dart

```diff
@@ -388,6 +386,45 @@ enum E {
       expect(method1.documentationComment, '/// Doc comment.');
     });
 
+    test("an enhanced enum's constructors are documented", () async {
+      var library = await bootPackageWithLibrary('''
+enum E {
+  one.named(1),
+  two.named(2);
+
+  final int x;
+
+  /// A named constructor.
+  const E.named(this.x);
+}
+''');
+      var namedConstructor =
+          library.enums.named('E').constructors.named('E.named');
+
+      expect(namedConstructor.isFactory, false);
+      expect(namedConstructor.fullyQualifiedName, 'enums.E.named');
+      expect(namedConstructor.nameWithGenerics, 'E.named');
+      expect(namedConstructor.documentationComment, '/// A named constructor.');
+    });
+    test("has an 'index' getter which can be referenced", () async {
```

Comment:
The test name does not look to correspond to the content.

Reply:
Thanks; done.